static/js/compose.js

```diff
@@ -784,6 +785,26 @@ function validate_private_message() {
         );
         return false;
     }
+
+    const deactivated_users_ids = [];
+    for (const user_id of user_ids) {
+        if (!people.is_person_active(user_id)) {
+            deactivated_users_ids.push(user_id);
+        }
+    }
+
+    if (deactivated_users_ids.length !== 0) {
+        const user_id = deactivated_users_ids[0];
+
+        context = {email: people.get_by_user_id(user_id).email};
+        compose_error(
+            $t_html({defaultMessage: "The user {email} has been deactivated"}, context),
+            $("#private_message_recipient"),
+        );
+
+        return false;
+    }
```

Comment:
I think it might read more cleanly to just do this `return` inside the above `for` loop, since we don't display more than one.

Reply:
Okay sure. Didn't noticed this much. updating this.